pemFioi/quickAlgo/interface-mobileFirst.js

```diff
@@ -238,6 +250,7 @@ var quickAlgoInterface = {
                         this.strings.reloadProgram +
                     "</div>" +
                     "<div rel='edit' class='item' onclick='quickAlgoInterface.editorBtn(\"edit\");'><span class='fas fa-pencil-alt'></span>" + this.strings.editButton + "</div>" +
+                    "<div rel='about' class='item' onclick='quickAlgoInterface.editorBtn(\"about\");'><span class='fas fa-question-circle'></span>" + this.strings.about + "</div>" +
```

Comment:
About button should be last, it's odd to have it in the middle.

Reply:
In the next version, the about has been placed last.